sonaranalyzer-dotnet/src/SonarAnalyzer.VisualBasic/Rules/PropertiesAccessCorrectField.cs

```diff
@@ -50,27 +50,39 @@ protected override IEnumerable<FieldData> FindFieldAssignments(IPropertySymbol p
 
             // we only keep information for the first location of the symbol
             var assignments = new Dictionary<IFieldSymbol, FieldData>();
+            FillAssignments(setter, true);
+            // If there're no candidate variables, we'll try to inspect one local method invocation with value as argument
+            if (assignments.Count == 0
+                && SingleInvocation(setter) is ExpressionSyntax expression
```

Comment:
SingleInvocation always returns ExpressionSyntax. Can use `{ }` if you want to check for null and assign a name to the return value.